.drone.yml

```diff
@@ -4594,10 +4594,10 @@ volumes:
   name: docker
 ---
 clone:
-  retries: 3
```

Comment:
Is that intentional? 🤔

Reply:
This is weird... let me check where that's coming from :/